### ISSUE TITLE
Distributor suggested EZ Map improvements

### ISSUE THREAD

**Issue by NejaE** [UbiquityRobotics/ubiquity_main#284]
We've reached out to one of our distributors to get some feedback about our offering and to support them in their sales better. They've provided the following feedback:

_On the EZ Map Pro software, it is a good kickstart to customer with programmer but alone, it cannot perform some basic task.

For example, to add below features, what would be the price needed?

    To add STOP BUTTON on the touchscreen (able to enable or disable)
    To add CONTINUE BUTTON on the touchscreen.(to enabled if  STOP BUUTON is enabled)
    To have the existing DELAY setting to continue moving if not STOP BUTTON is pressed.
    To have the existing setting to output Voice if enabled at particular position (This feature need fine tuning as voice output not clear, etc…)
    To have setting with above to allow user to define if STOP BUTTON pressed but if XX minute, no CONTINUE pressed, it will auto move to next position.

We think that with above features added, EZ Map can perform basic task._

@MoffKalast Please review and let us know if any of these upgrades are feasible or even make sense long term.

**Comment by MoffKalast** [comment]
Yes, that's been a known issue for a while. Right now the rough plan is to modify and integrate the existing conveyorbot touchscreen to enable that sort of functionality. I think the best option may be to allow adding/binding touchscreen buttons to custom actions in the web part, then people can just trigger anything they need. Adam's plugin rework may need to to come before that though, so it would make sense to get something done quick and dirty for testing in the meantime.

We should probably log this as an issue on one of the ezmap repos instead of main.
